### ISSUE TITLE
Iphone behaviour strange

### ISSUE THREAD

**Issue by lashan-gamelounge** [usablica/intro.js#2080]
## Description
When I open one of the examples from the tours section on the iPhone 15 pro, initially it loads when I try to go next it will have only overlay and no highlighted section.

## Expected Behavior
It should open the second highlighted modal with the tooltip

## Actual Behavior
initially, it loads when I try to go next it will have only overlay and no highlighted section on the iPhone 15 pro

## Errors and Screenshots (optional)
![IMG_7175](https://github.com/user-attachments/assets/76744dd9-3dc3-48ef-abd8-b5ed15b55dd2)
![IMG_7176](https://github.com/user-attachments/assets/76b2db07-a546-4211-95ff-1b54986838ef)## Example (recommended)
Try to open this https://introjs.com/docs/tour/examples/confirm-before-exit on iPhone 15 pro and hit the next button.

## Environment (optional)
iPhone 15 pro, chrome

**Comment by chris2k20** [comment]
Same here, the seconds screen looks very bad. I have to disable intro.js on mobile devices because Safari and Chrome are not working at the moment. (Used that: https://cdnjs.cloudflare.com/ajax/libs/intro.js/7.2.0/intro.min.js) 
How can we fix this? 
By the way, I really love your project. So thank you very much for the time you spend on this project. 🙏🏻

**Comment by higege-amdeker** [comment]
Perhaps the problem is dependent on iOS 18.0?
It seems to work fine with iOS 17.5.

**Comment by chris2k20** [comment]
Yes, I have the latest iOS beta dev version. What do you need?

**Comment by Fant0m771** [comment]
Hi, I have the same behavior for the desktop Safari, the tooltip does not appear before you scroll the page, after scroll event is happened the tooltip appeared. Safari browser Version 18.0.1 (19619.1.26.111.11, 19619), MacOS 14.7

**Comment by sjoerdbooij** [comment]
I also experience the same issue on my iPhone 13 in Chrome with iOS 18.1.1. The first tooltip is fine, but after pressing NEXT the tooltips are no longer there, just highlighting the element.

**Comment by mithlesh425** [comment]
+1 for me, not working for iPhone iOS 18 beta.

**Comment by TR-Emre** [comment]
> I also experience the same issue on my iPhone 13 in Chrome with iOS 18.1.1. The first tooltip is fine, but after pressing NEXT the tooltips are no longer there, just highlighting the element.

Yes. I have the same problem. There is no problem on PC and Android devices, but on iOS devices, after the first step, the tips are no longer visible. In addition, the area is not clickable.

1) ![https://github.com/user-attachments/assets/27337d9e-c6ad-4e10-b195-437467b1e481](https://github.com/user-attachments/assets/27337d9e-c6ad-4e10-b195-437467b1e481)

2) ![https://github.com/user-attachments/assets/13eb6228-18a4-41dc-bdb8-62bcd1c9debe](https://github.com/user-attachments/assets/13eb6228-18a4-41dc-bdb8-62bcd1c9debe)

**Comment by lch2273146069** [comment]
I also encountered the same problem when using it in the project and hope it can be resolved
